Library/Homebrew/extend/os/mac/caveats.rb

```diff
@@ -37,7 +37,9 @@ def plist_caveats
 
     if f.plist_manual || f.service?
       command = if f.service?
-        f.service.command.join(" ")
+        f.service.command
+         .map { |arg| (arg =~ /\s/) ? "'#{arg}'" : arg } # wrap multi-word arguments in quotes
+         .join(" ")
```

Comment:
> are you referring to the chained syntax?

I think this is what @MikeMcQuaid is referring to (but correct me if I'm wrong)

```suggestion
        f.service.command
          .map { |arg| (arg =~ /\s/) ? "'#{arg}'" : arg } # wrap multi-word arguments in quotes
          .join(" ")
```

Although an alternative that I think would still align with our conventions that you may find more aesthetically appealing might be

```suggestion
        f.service.command.map do |arg|
          return "'#{arg}'" if arg.match?(/\s/) # wrap multi-word arguments in quotes

          arg
        end.join(" ")
```